docs/concepts/protocol/02-stream-types.mdx

```diff
@@ -242,30 +235,43 @@ The Timelock shape locks all tokens for a specified period. Users cannot access
   }}
 />
 
-Once the set period elapses, the full amount becomes accessible to the recipient. This setup is particularly
-advantageous for projects seeking to stabilize token pricing and minimize market volatility, encouraging investors to
-maintain their stake over a more extended period.
+Initially, a set amount of tokens are made available to the recipient as an upfront payment. After this initial unlock,
+the tokens are held during the cliff period until the moment of time set. The release resumes in a linearly post-cliff.
 
 :::info
 
-The gas cost for this shape is approximately _239,232_ on Mainnet. This may vary due to multiple factors.
+The gas cost to create this shape is approximately _290,477_ on Mainnet. This may vary due to multiple factors.
 
 :::
 
-### Unlock Linear
+## Lockup Tranched
 
-The Unlock Linear shape combines an initial immediate release of tokens with a steady, linear payout over time. This
-shape is ideal for employment contracts that include a signing bonus along with a regular payout schedule.
+Lockup Tranched streams are, as the name says, streams that have token unlocks in tranches. Even though you can use
+Lockup Dynamic to create a traditional vesting structure with periodic unlocks, Lockup Tranched is specifically design
+for that use case. As a result, a stream with tranches created using Lockup Tranched is more gas efficient than the same
+stream created using Lockup Dynamic.
+
+These streams are powered by a number of user-provided "tranches", which is covered in the tranches article.
```

Comment:
Let's link to the tranches article here.

Reply:
https://github.com/sablier-labs/docs/pull/138/commits/bffb33ee451c19488d0d603bc30fb7d5619bb106